analytic_amount_security/security/security.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data noupdate="0">
+        <!-- Change the implied ids to add constrain more or less the visibility scope -->
+        <record id="group_allow_read_analytic_amount" model="res.groups">
+            <field name="name">Allow to read analytic costs</field>
+            <field name="category_id" ref="base.module_category_hidden" />
+        </record>
+    </data>
+    <data noupdate="1">
+        <record id="base.group_user" model="res.groups">
+            <field
+                name="implied_ids"
+                eval="[(4, ref('account.group_show_line_subtotals_tax_excluded'))]"
+            />
+        </record>
+        <record id="analytic.group_analytic_accounting" model="res.groups">
+            <field
+                name="implied_ids"
+                eval="[(4, ref('account.group_show_line_subtotals_tax_excluded'))]"
```

Comment:
question: 🤔 I don't understand why the relationship with `account.group_show_line_subtotals_tax_excluded`. 

It's probably buggy anyway, as seen in the above comment. However, in any case, this is a hidden group that is applied by default [when configuring Odoo for B2B or B2C](https://github.com/odoo/odoo/blob/239a98f578759909f4920fa327ea4f67583b45ad/addons/account/models/res_config_settings.py#L210-L214):

![image](https://github.com/user-attachments/assets/4915aaaa-1136-4e61-a140-0a2027865207)

So what's the relationship between this group and the fact of displaying or not the analytic amount?